modules/version-update/pages/upgrade-from-community-to-a-subscription-edition.adoc

```diff
@@ -82,9 +85,39 @@ Replace __<platform-setup-install-folder>__ with the folder where you want to in
 Downloading the `platform-setup-sp` tool is optional as it is already packaged in the bundle archive when building your SCA using the `bundle` mode. It is recommended when using a Docker image.
 ====
 
+==== Upgrade a Bonita Runtime with a Tomcat bundle
+
+To upgrade a Bonita Runtime from a Community edition to an Enterprise edition, follow these steps:
+
+. Download and unzip the Bonita Enterprise bundle to `<bonita-enterprise-installation>` folder
+. Copy your database.properties fom your Community installation `setup` folder to `<bonita-enterprise-installation>/setup` folder
+. Run `<bonita-enterprise-installation>/setup/setup(.sh/.bat) pull` to fetch your current Community configuration in `<bonita-enterprise-installation>/setup/platform_conf/current`
+. Retrieve an xref:ROOT:licenses.adoc[Enterprise license] file
+. Copy your license file into `<bonita-enterprise-installation>/setup/platform_conf/licenses`
+. Push your new license in the database: `<bonita-enterprise-installation>/setup/setup(.sh/.bat) push`
+. Stop Bonita Community platform with `<bonita-community-installation>/stop-bonita(.sh/.bat)` script.
+. Start the Enterprise Bonita Runtime using `<bonita-enterprise-installation>/start-bonita(.sh/.bat)` script
+
+Once you have started Bonita Enterprise Runtime at least once, you can pull your Enterprise configuration (`<bonita-enterprise-installation>/setup/setup(.sh/.bat) pull`)
+and change the Enterprise-specific configuration files to match your needs. See xref:runtime:bonita-platform-setup.adoc[Bonita Runtime setup tool] for details about
+how to apply configuration changes.
+
+===== Import Enterprise Admin App
+
+Starting with Bonita 2021.2, Bonita Portal has been replaced with Bonita Applications.
+In Enterprise edition, an additional application for administrators is available for manual import - Bonita Administrator Application.
+
+Here are the steps to follow :
+
+. Go to `<bonita-enterprise-installation>/server/webapps/bonita/WEB-INF/classes/org/bonitasoft/web/application/`.
+. Unzip the file `bonita-admin-application-sp-x.y.z.zip`. You should have a xml and a png file.
+. Connect to the bonita platform using a super administrator user.
+. Go to applications, import the xml file.
+. ⚠ (Optionally, click on more details of your newly imported application, and import the png file as logo)
+
 ==== Remove Community Admin App
 
-If you chose to include the Admin App in your SCA, you will have two versions of the app installed: the community and enterprise versions. To avoid any confusion, you can remove the community version from the `Applications` menu of the Super Admin application.
+If you chose to include the Admin App in your SCA or you were using the Bundle, you will have two versions of the app installed: the community and enterprise versions. To avoid any confusion, you can remove the community version from the `Applications` menu of the Super Admin application.
```

Comment:
```suggestion
If you chose to include the Admin App in your SCA or if you were using the Bundle, you will have two versions of the app installed: the community and enterprise versions. To avoid any confusion, you can remove the community version from the `Applications` menu of the Super Admin application.
```